### ISSUE TITLE
Mapping for 054c-0268-Sony PLAYSTATION(R)3 Controller Chrome Windows NT

### ISSUE THREAD

**Issue by rhmoller** [ericlathrop/html5-gamepad#13]
Please include this mapping:

```
{
  "axes": {
    "dpad x": {
      "index": 6
    },
    "dpad y": {
      "index": 7
    },
    "left stick x": {
      "index": 0
    },
    "left stick y": {
      "index": 1
    },
    "right stick x": {
      "index": 2
    },
    "right stick y": {
      "index": 3
    },
    "right trigger": {
      "index": 13
    },
    "left trigger": {
      "index": 12
    }
  },
  "buttons": {
    "a": {
      "index": 14
    },
    "dpad down": {
      "index": 6
    },
    "dpad left": {
      "index": 7
    },
    "dpad right": {
      "index": 5
    },
    "dpad up": {
      "index": 4
    },
    "left stick": {
      "index": 1
    },
    "left stick down": {
      "axis": 1,
      "direction": 1
    },
    "left stick left": {
      "axis": 0,
      "direction": -1
    },
    "left stick right": {
      "axis": 0,
      "direction": 1
    },
    "left stick up": {
      "axis": 1,
      "direction": -1
    },
    "right stick": {
      "index": 2
    },
    "right stick down": {
      "axis": 3,
      "direction": 1
    },
    "right stick left": {
      "axis": 2,
      "direction": -1
    },
    "right stick right": {
      "axis": 2,
      "direction": 1
    },
    "right stick up": {
      "axis": 3,
      "direction": -1
    },
    "y": {
      "index": 12
    },
    "left shoulder": {
      "index": 10
    },
    "left trigger": {
      "index": 8
    },
    "right trigger": {
      "index": 9
    },
    "right shoulder": {
      "index": 11
    },
    "x": {
      "index": 15
    },
    "b": {
      "index": 13
    },
    "start": {
      "index": 3
    },
    "back": {
      "index": 0
    },
    "home": {
      "index": 16
    }
  },
  "name": "054c-0268-Sony PLAYSTATION(R)3 Controller Chrome Windows NT",
  "supported": [
    {
      "browser": "Chrome",
      "id": "054c-0268-Sony PLAYSTATION(R)3 Controller",
      "os": "Windows NT"
    }
  ]
}
```

**Comment by rhmoller** [comment]
This is Firefox 53 on Ubuntu 17.04

**Comment by ericlathrop** [comment]
Merged and published as 1.1.0. Thanks!
